cmd/install.go

```diff
@@ -142,13 +156,28 @@ func install() {
 		askBecomePassFlag = "-K"
 	}
 
+	// Set the SSL flag if cert and key are defined
+	var sslCertKeyFlag string
+	if sslCert != "" && sslKey != "" {
+	sslCertAbs, err := filepath.Abs(sslCert)
+	if err != nil {
+		check(errors.New("Unable to get absolute path of " + sslCert))
+	}
+	sslKeyAbs, err := filepath.Abs(sslKey)
+	if err != nil {
+		check(errors.New("Unable to get absolute path of " + sslKey))
+	}
+		sslCertKeyFlag = fmt.Sprintf("-v %s:/runner/certs/quay.cert -v %s:/runner/certs/quay.key", sslCertAbs, sslKeyAbs)
+	}
+
 	// Run playbook
 	log.Printf("Running install playbook. This may take some time. To see playbook output run the installer with -v (verbose) flag.")
 	podmanCmd := fmt.Sprintf(`sudo podman run `+
 		`--rm --interactive --tty `+
 		`--workdir /runner/project `+
 		`--net host `+
 		imageArchiveMountFlag+ // optional image archive flag
+		sslCertKeyFlag + // optional ssl cert/key flag
```

Comment:
Similar request here, for consistency can the whitespace between `sslCertKeyFlag` and the `+` be removed?